### PR TITLE
Upgrade to GitLab CE 15.7.0

### PR DESCRIPTION
**Pull request by antt1995**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+FROM ubuntu:focal-20221130
 
-ARG VERSION=15.6.2
+ARG VERSION=15.7.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=3.0.4 \
     RUBY_SOURCE_SHA256SUM="70b47c207af04bce9acea262308fb42893d3e244f39a4abc586920a1c723722b" \
     GOLANG_VERSION=1.18.8 \
     GITLAB_SHELL_VERSION=14.13.0 \
```
Comment (sachilles) [inline Dockerfile:9]: This should be updated to 14.14.0 (see <https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v15.7.0/GITLAB_SHELL_VERSION>).

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,14 @@
-FROM ubuntu:focal-20221019
+FROM ubuntu:focal-20221130
 
-ARG VERSION=15.6.2
+ARG VERSION=15.7.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=3.0.4 \
     RUBY_SOURCE_SHA256SUM="70b47c207af04bce9acea262308fb42893d3e244f39a4abc586920a1c723722b" \
```
Comment (sachilles) [inline Dockerfile:7]: I guess we can update to ruby 3.0.5 (see <https://gitlab.com/gitlab-org/gitlab-omnibus-builder/-/blob/master/docker/VERSIONS>).

Please change to
```
RUBY_VERSION=3.0.4 \
RUBY_SOURCE_SHA256SUM="9afc6380a027a4fe1ae1a3e2eccb6b497b9c5ac0631c12ca56f9b7beb4848776"
```
(see <https://www.ruby-lang.org/de/downloads/>).

File: Changelog.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+- gitlab: upgrade CE to v15.7.0
+- gitaly: upgrade to v15.7.0
+- gitlab-pages: upgrade to v15.7.0
+
+**15.6.3**
+- gitlab: upgrade CE to v15.6.3
+- gitaly: upgrade to v15.6.3
+- gitlab-pages: upgrade to v15.6.3
+- ubuntu: upgrade to focal-20221130
+
```
Comment (sachilles) [inline Changelog.md:15]: Please delete this empty line.

**Review by sachilles — CHANGES_REQUESTED** [review]
Could you be so kind and incorporate the requested changes?

File: Changelog.md (hunk 3 above)
Comment (sachilles) [reply Changelog.md:15]: Please do not forget the entries for the requested changes.

**Comment by antt1995** [comment]
@sachilles As requested

**Review by sachilles — APPROVED** [review]
LGTM

**Comment by sachilles** [comment]
@antt1995 Thank you for the quick response and for your contribution.

**Pull request merged**